btree/btree.c

```diff
@@ -2028,7 +2018,7 @@ static void ff_init(const struct nd *node, int shift, int ksize, int vsize,
 	h->ff_shift = shift;
 	h->ff_ksize = ksize;
 	h->ff_vsize = vsize;
-	mem_update(node, tx, h, sizeof *h);
+	//m0_be_tx_capture();
```

Comment:
Revert to C-style comments. Also write a TBD with a few words indicating that tx_capture needs to happen at this location. Do the same for all the other places where you think the transaction capture should happen.

Reply:
Updated in the latest commit.